crates/fluvio-controlplane/src/lib.rs

```diff
@@ -10,3 +10,5 @@ mod alias {
 
     pub type PartitionMetadata<C> = MetadataStoreObject<PartitionSpec, C>;
 }
+
+pub const CONSUMER_STORAGE_TOPIC: &str = "fluvio-consumers";
```

Comment:
`fluvio` prefix is kind of redundant.   Something like: `consumer-offset` ?

Reply:
Renamed